WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -17,9 +17,18 @@ def index
     paginate json: competitions
   end
 
+  COMPETITION_INFO_SERIALIZE_OPTIONS = {
+    only: %w[id name website start_date registration_open registration_close announced_at cancelled_at end_date competitor_limit extra_registration_requirements enable_donations refund_policy_limit_date event_change_deadline_date waiting_list_deadline_date on_the_spot_registration on_the_spot_entry_fee_lowest_denomination qualification_results event_restrictions base_entry_fee_lowest_denomination currency_code allow_registration_edits allow_registration_self_delete_after_acceptance allow_registration_without_qualification refund_policy_percent use_wca_registration guests_per_registration_limit venue contact force_comment_in_registration use_wca_registration external_registration_page guests_entry_fee_lowest_denomination guest_entry_status information],
+    methods: %w[url website short_name city venue_address venue_details latitude_degrees longitude_degrees country_iso2 event_ids registration_opened? main_event_id number_of_bookmarks using_stripe_payments? uses_qualification? uses_cutoff? events_with_rounds schedule_wcif],
+    include: %w[delegates organizers tabs],
+  }.freeze
```

Comment:
This slightly reminds me of https://github.com/thewca/wcif/pull/17.
What information is there in this big array that you need, but that you cannot currently get through WCIF?

Reply:
I changed it to only have the new fields that I need, but as mentioned above, most of them are not in the wcif